core/trino-main/src/main/java/io/trino/connector/CatalogPruneTask.java

```diff
@@ -137,8 +137,8 @@ public void pruneWorkerCatalogs()
         List<CatalogHandle> activeCatalogs = getActiveCatalogs();
         pruneWorkerCatalogs(online, activeCatalogs);
 
-        // prune all inactive catalogs - we pass an empty set here because manager always retains active catalogs
-        connectorServicesProvider.pruneCatalogs(ImmutableSet.of());
+        // prune inactive catalogs locally
```

Comment:
sorry I don't get this one, comment says prune inactive catalogs but you pass activeCatalogs. I don't know how it is supposed to work :/